common-docs/identity/sign-in.md

```diff
@@ -0,0 +1,35 @@
+# Microsoft Accounts and MakeCode
+
+**Note**: This feature only available in [**MakeCode Arcade**](https://arcade.makecode.com).
+
+## What is a Microsoft Account?
+
+A Microsoft Account, or MSA, is the personal user account you create when you want to sign into Microsoft services like outlook.com, Xbox, Skype, etc. Read more [here](https://aka.ms/AAdd6f8).
+
+## How does it work in MakeCode?
+
+A number of new features are coming that will only be available to signed-in users. These features need to know your identity so they can do things for you like store data in the cloud, and connect you to other users. The first of these features is one we're calling [Cloud Sync](/identity/cloud-sync). With Cloud Sync, when you're signed into MakeCode, we'll store your projects in the cloud where they're accessible from anywhere.
+
+## Is it required?
+
+No. Sign in only if you want to take advantage of features like Cloud Sync.
+
+## Does it cost money?
+
+Nope! Creating an MSA account is free. Using MakeCode is also free. As a reminder, MSA accounts work across a range of Microsoft products and services, so you can use it outside MakeCode. Some of these products and services may not be free (Minecraft Marketplace content, for example), but this is outside the MakeCode ecosystem.
+
+## What will other users see?
+
+Your signed in status is visible only to you. In the future we may add features that allow others to see your presence. When that time comes, we would ask you to create a MakeCode Profile so that your name and picture are decoupled from your MSA profile.
+
+## Do I already have a Microsoft Account?
+
+It is possible! Do you play Minecraft, games on an Xbox, subscribe to Office 365, or use any other Microsoft products or services? If so, you probably already have a Microsoft Account.
+
+## Does this mean I have to create a new email address?
+
+No. You can create an MSA using your current email address if it is not hosted by Microsoft (e.g.: GMail, Yahoo Mail, and others). When creating your MSA account, just enter your existing email address.
```

Comment:
I think it should be styled as "Gmail" (at least that's what it looks like on their docs https://developers.google.com/gmail/design/css)